frontend/src/tests/components/BasicCourseSearch/BasicCourseSearchForm.test.js

```diff
@@ -147,18 +141,14 @@ describe("BasicCourseSearchForm tests", () => {
       total: 0,
     };
 
-    const setCourseJSONSpy = jest.fn();
     const fetchJSONSpy = jest.fn();
```

Comment:
duplicate declaration on line 98

Reply:
they're 2 different tests

Reply:
is this supposed to be a redeclaration or can this be removed

Reply:
we are mocking a function in two separate tests. Then, in the tests we call expect(fetchJSON) .toBeCalled(). We need to have a separate variable for each test (that is why we have 2) because, if say the function was called in Test 1 but not Test 2, the expect() call in Test2 will still be true because we have called the jest function in the previous test.

So yes it is supposed to be there